yarn.lock

```diff
@@ -600,12 +600,7 @@
     "@nodelib/fs.scandir" "2.1.5"
     fastq "^1.6.0"
 
-<<<<<<< HEAD
 "@nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers", hardhat-deploy-ethers@^0.3.0-beta.10:
-  name "@nomiclabs/hardhat-ethers"
-=======
-"@nomiclabs/hardhat-ethers@npm:hardhat-deploy-ethers@^0.3.0-beta.10", hardhat-deploy-ethers@^0.3.0-beta.10:
->>>>>>> 90d5f5e... libs/utils: rework utilities and update compiler version and openzeppelin version
```

Comment:
I'm surprised that this conflict annotation didn't break CI before.

Reply:
Actually the CI is failing because the BondingManager contract was updated but the corresponding test cases were not.

Reply:
Yep noticed that. Was just noting here that I was surprised that this particular conflict annotation didn't cause problems on its own. Would've expected it to mess up the yarn install 🤷

Reply:
yeah, this is surprising to me too

I was trying to reply to the previous comment `missing property on a type`. Somehow ended up replying on this thread 😅